tools/reflection_generator/reflection_generator.py

```diff
@@ -108,37 +107,56 @@ def Touch(path):
 
 def main(argv):
   usage = """Usage: %prog [OPTIONS]
-This script can generate bridge and wrap source files for given directory. 
+This script can generate bridge and wrap source files for given directory.
 \'input_dir\' is provided as directory containing source files.
   """
   option_parser = optparse.OptionParser(usage=usage)
   option_parser.add_option('--input_dir',
-                           help= ('Input source file directory which contains'
-                                  'input files'))
+                           help=('Input source file directory which contains '
+                                 'input files'))
+  option_parser.add_option('--template_dir',
+                           help=('Templates directory to generate java source '
+                                 'file'))
   option_parser.add_option('--bridge_output',
-                           help=('Output directory where the bridge code'
-                                 'is placed.'))
-  option_parser.add_option('--wrap_output',
-                           help=('Output directory where the wrap code'
-                                'is placed.'))
-  option_parser.add_option('--helper_class',
-                           help=('the path of ReflectionHelper java source, '
-                                'will copy it to output folder'))
+                           help=('Output directory where the bridge code is '
+                                 'placed.'))
+  option_parser.add_option('--wrapper_output',
+                           help=('Output directory where the wrap code is '
+                                 'placed.'))
   option_parser.add_option('--stamp', help='the file to touch on success.')
+  option_parser.add_option('--target_arch', help='Target Architecture')
+  option_parser.add_option('--sdk_version', help='API Version')
+  option_parser.add_option('--min_sdk_version', help='Min API Version')
```

Comment:
It is not common practice to use `_` as option, instead we should use `-`. I know this is not this PR fault, but let's change that.